docs/cep/table/index.md

```diff
@@ -59,17 +72,32 @@ The following parameters are configured in a table definition:
 | attribute type   | The type of each attribute defined in the schema.  This can be `STRING`, `INT`, `LONG`, `DOUBLE`, `FLOAT`, `BOOL`, or `OBJECT`.     |
 | from        | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a source node of the edge.      | _from         | STRING              | Yes      |
 | to          | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a destination node of the edge. | _to      | STRING              | Yes      |
+| [WITH (property_name = expression [, ...])] | Optional properties for the new table, such as a time-to-live or a partition key. |
+| SELECT select_expr [, ...] | The selection criteria for the new table. |
+| FROM from_stream … | The name of the existing stream to select data from. |
 
-## Examples
+## Example 1
 
 The following defines a local table named `RoomTypeTable` with `roomNo` and `type` attributes of data types `int` and `string` respectively.
 
 ```sql
 CREATE TABLE RoomTypeTable (roomNo int, type string);
 ```
 
+## Example 2
+
 The following defines a global table named `SweetProductionCollection` with `name` and `amount` attributes of data types `string` and `double`.
 
 ```sql
 CREATE TABLE GLOBAL SweetProductionCollection (name string, amount double);
 ```
+
+## Example 3
+
+```sql
+CREATE TABLE StockTable (symbol string, price float, volume long)
+AS SELECT symbol, price, volume
```

Comment:
@diana-macrometa I forget if we discussed this, but was there a recent change that added `AS` to the `SELECT` statement?  I recall these statements simply being `SELECT` but I trust that your information is probably more recent!